README.md

```diff
@@ -32,7 +32,7 @@ contributors as reviewers to the PR.
 4. Organize offline review or/and bring the RFC to [UXL Foundation Math SIG forum](https://lists.uxlfoundation.org/g/Math-SIG), [UXL Foundation Open Source Working Group](https://lists.uxlfoundation.org/g/open-source-wg), or any other related forums in order to collect feedback.
     * It's recommended to keep all feedback as part of PR review, so it also
 will be documented in one place
-5. If changes affect API defined by [oneMKL specification](https://oneapi-spec.uxlfoundation.org/specifications/oneapi/latest/elements/onemkl/source/) the design document must be reviewed by [UXL Foundation Math SIG forum](https://lists.uxlfoundation.org/g/Math-SIG) and contributed to [oneAPI specification](https://github.com/uxlfoundation/oneAPI-spec), and only after it the proposed changes can be implemented in this project.
+5. If the changes affect the API defined by [oneMKL specification](https://oneapi-spec.uxlfoundation.org/specifications/oneapi/latest/elements/onemkl/source/) a new [issue](https://github.com/uxlfoundation/oneAPI-spec/issues) with the \[RFC\] tag must be created to describe the API changes. The specification RFC must be reviewed by [UXL Foundation Math SIG forum](https://lists.uxlfoundation.org/g/Math-SIG), and only after it the proposed changes can be implemented in this project.
```

Comment:
I didn't find `RFC` in the current [labels](https://github.com/uxlfoundation/oneAPI-spec/labels). Will it be created? Or does it mean the issue title should be prefixed by "[RFC]"?

nitpicking
```suggestion
5. If the changes affect the API defined by [oneMKL specification](https://oneapi-spec.uxlfoundation.org/specifications/oneapi/latest/elements/onemkl/source/), a new [issue](https://github.com/uxlfoundation/oneAPI-spec/issues) with the \[RFC\] tag must be created to describe the API changes. The specification RFC must be reviewed by [UXL Foundation Math SIG forum](https://lists.uxlfoundation.org/g/Math-SIG), and only after it the proposed changes can be implemented in this project.
```

Reply:
It's actually not a label, but a tag in the issue name https://github.com/uxlfoundation/oneAPI-spec/issues?q=is%3Aissue+is%3Aopen+RFC but maybe we can ask for the introducing the new label in the spec.